component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1522,4 +1535,29 @@ private void setUserClaimValues(AuthenticatedUser authenticatedUser, Map<String,
             throw new AuthenticationFailedException(errorMessage, e);
         }
     }
+
+    private String getUnlockTime(AuthenticatedUser authenticatedUser)
```

Comment:
Does this return update time in milliseconds? If so shall we convey that in the method name?

Reply:
fixed in f3b362c